app/action/interventionList/post.controller.js

```diff
@@ -0,0 +1,36 @@
+const { body } = require('express-validator')
+const { countWords } = require('../../../common/utils/util')
+const { logger } = require('../../../common/logging/logger')
+
+const wordsAllowed = 50
+
+const validationRules = () => {
+  return [
+    body('actionType')
+      .isIn(['description', 'intervention'])
+      .withMessage('Select an action'),
+    body('description')
+      .if(body('actionType').equals('description'))
+      .isLength({ min: 1 })
+      .withMessage('Describe the action'),
+    body('description')
+      .if(body('actionType').equals('description'))
+      .custom(value => {
+        return countWords(value) <= wordsAllowed
+      })
+      .withMessage(`Action description must be ${wordsAllowed} words or fewer`),
+    body('description')
+      .trim()
+      .escape(),
+    body('intervention')
+      .if(body('actionType').equals('intervention'))
+      .matches(/^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[0-9a-f]{12}$/, 'i')
```

Comment:
Should the front end care about what the intervention ID format is?

Reply:
Arguable; the data type is a uuid, so how loose or heavy do you want to be with it? I've taken the  heavier perspective, that the post requires a UUID, so thats what I send.

Reply:
I think the argument is that we're only passing on what's been given to us, we're not generating it, using it directly in a way that depends on the format or mutating it - we're not ultimately responsible for the format of the UUID

Reply:
Fair point. As is, the regexp means we don't have to sanitise too. Following discussion, leaving in for now and resolving.